vm.py

```diff
@@ -107,3 +107,12 @@ def poweroff():
   from nagios import schedule_downtime
   execute(schedule_downtime, env['host_string'])
   run("sudo poweroff")
+
+@task
+@hosts('puppetmaster-1.management')
+def host_key(hostname):
+  """Check the SSH host key of a machine"""
+  temp_file = '/tmp/ssh_host_key'
```

Comment:
It doesn't look like `ssh-keygen` is happy to receive things that aren't files, but a bit of Googling has revealed that 3 left chevrons works.